sql/smoot_clients_last_seen_1percent_raw_v1.init.sql

```diff
@@ -0,0 +1,19 @@
+CREATE
+OR REPLACE
+TABLE
+  smoot_clients_last_seen_1percent_raw_v1
+PARTITION BY
+  submission_date AS
+SELECT
+  DATE(NULL) AS submission_date,
+  CURRENT_DATETIME() AS generated_time,
+  * EXCEPT (submission_date_s3),
+  0 AS days_seen_bits,
+  0 AS days_visited_5_uri_bits,
+  0 AS days_opened_dev_tools_bits,
+  0 AS days_since_created_profile
```

Comment:
I've switched to storing each usage criterion as an INT64 representing usage on each of the previous 28 days, as pioneered by @relud. I was pushing back on it, but you've given me the courage to understand the available operators better and now I see the beauty.

With judicious use of comments and factoring some of the bit manipulation logic into UDFs, I do think we can make this maintainable.

Also of note, this is now a last_seen_raw table, and we delay calculating `days_since_*` to a view on top of the raw table.